### DISCUSSION TITLE
Why the change from the cog wheel to hyperlink for notifications?

### DISCUSSION THREAD

**Discussion by bugz8unny69** [community/community#21688] [New to GitHub]
<p>Hello everyone,</p>
<p>I came back yesterday after a brief hiatus to my dismay that the cog wheel next to subscribe button went missing, initially I had assume that the feature was reverted but to noticed that it had moved to a ‘Customize’ hyperlink. Why was this change implemented? It was a very clear workflow to, at least to me, to something slightly convoluted?</p>
<p>Regards</p>

**Comment by lee-dohm** [comment]
<p>I assume by “Subscribe” and “Customize” you’re talking about the button and link here in the issue or PR sidebar?</p>
<p><em><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/9/9f3a7b99d03670870cc61dbbda50199641d32431.png" data-download-href="/uploads/short-url/mIBdqPg1S5zD6CZJGdcuKPUHqqR.png?dl=1" title="Screen Shot 2019-07-01 at 11.27.31 AM.png"><img src="https://user-images.githubusercontent.com/1038121/181066357-c8313e43-f495-4ea9-b66f-21c65d2360c6.png" alt="Screen Shot 2019-07-01 at 11.27.31 AM.png" data-base62-sha1="mIBdqPg1S5zD6CZJGdcuKPUHqqR" width="578" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/9/9f3a7b99d03670870cc61dbbda50199641d32431_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-07-01 at 11.27.31 AM.png</span><span class="informations">578×1820 95.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div></em></p>
<p>I’ll pass along the feedback that you found the change confusing to the product team. Thanks for the feedback!</p>
<p>As far as why the change was made, I don’t have any information on that. If I had to guess though, the things that still have a gear icon are things that configure the repository or the issue, whereas the “Subscribe” and “Customize” configure  <strong>your</strong> notifications of that issue or PR.</p>
<p>I hope that helps!</p>

**Comment by bugz8unny69** [comment]
<p>Thanks for the reply and yes that’s exactly what I’m referring to. It’s one of those things where you are use to a specific workflow and then things get moved around. So far I’m getting accustom to the new layout but I still prefer the old way. It’s easy to find for my bad eyes. If you guys can make changing it faster and more efficient, I will be forever grateful. I go through 100s of E-Mails a day and I user ‘notify when it’s closed’ 100s of times during that period.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>&gt;  If you guys can make changing it faster and more efficient, I will be forever grateful. I go through 100s of E-Mails a day and I user ‘notify when it’s closed’ 100s of times during that period.</p>
<p>This is great feedback! I’ll pass it along :+1: Thanks very much for sharing your pain points with the system.</p>
